src/elements/frame-router.ts

```diff
@@ -129,11 +130,11 @@ export default class FrameRouterElement extends HTMLElement {
    */
   public attributeChangedCallback(
     name: string,
-    oldValue: string,
-    newValue: string
+    oldValue: unknown,
+    newValue: unknown
```

Comment:
`unknown` is not the correct type here. I think `string | null` is accurate, or at least accurate enough.  I don't think browsers will pass undefined here. If you wanted to account for that, something like `oldValue?: string | null` would work fine, although I really don't think it's necessary.  `unknown` indicates we have no idea what it is, could be a number or an object or a function, which doesn't really apply here.